ec_api/apps/api_proxy/views.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
+from django.utils.module_loading import import_string
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from api_proxy.upstream_api_client import ResponseBuilderApiClient
+
+
+def get_upstream_client():
+    DEFAULT_CLIENT = "api_proxy.upstream_api_client.DCApiClient"
```

Comment:
Should this be set in the base settings? Then you wouldn't need to use `getattr`

Reply:
_could do_, but actually in this case I'm just using settings as a singleton object to pass some information around. There's actually no case where this would ever be changed, meaning it's not very helpful as a setting.